app/models/project.rb

```diff
@@ -44,6 +44,22 @@ def self.ransackable_associations(_auth_object = nil)
     %w[namespace]
   end
 
+  def accessible_from_namespace?(other_namespace) # rubocop:disable Metrics/AbcSize
```

Comment:
I feel like there is likely a better way to do this.

Reply:
Changed in https://github.com/phac-nml/irida-next/pull/897/commits/bb6e249f1cee578c751d4eabf5392026fd2e3ef8